src/helpers/assets.ts

```diff
@@ -282,107 +283,97 @@ export const buildBriefUniqueTokenList = (
   // group the assets by collection name
   const assetsByName = groupBy<UniqueAsset>(filteredUniqueTokens, token => token.familyName);
```

Comment:
Can do this as a followup, but most if not all of this logic could be consolidated into a single loop.

Reply:
This whole block:

https://github.com/rainbow-me/rainbow/blob/f7a4e2c2aa6ae04bc8bf58a9c18a8bf0a797bf32/src/helpers/assets.ts#L266-L284